### ISSUE TITLE
Display error messages to user when adding email address fails

### ISSUE THREAD

**Issue by superdump** [vector-im/riot-web#3557]
Stating only 'Unable to add email' is much less useful than giving the reason why from the error response.

**Comment by lukebarnard1** [comment]
In particular, the error that arises from an email address already being in use:

```
Unable to add email address luke@lukebarnard.co.uk M_THREEPID_IN_USE: This email address is already in use
```
This is logged, but the dialog ("Unable to add email address") is decidedly unhelpful.

**Comment by ara4n** [comment]
looks like i fixed this on Apr 23 in a2be764681240644ba5cbfaa7965adc308094dbf
